examples/clean-architecture/entities/note.entity.js

```diff
@@ -0,0 +1,40 @@
+var isEmptyString = function (str) {
+  return str === undefined || str === null || str.trim() === "";
+};
+var throwIfEmpty = function (str, field) {
+  if (isEmptyString(str))
+    throw new Error("field '" + field + "' should not be empty");
+};
+
+function Note(title, content, createdAt, updatedAt, id) {
```

Comment:
It could be preferable to use ES6 class, no?

Reply:
@Romakita sure! In fact, in my first version, I've used ES6 classes but (at least on the master branch) the linter tells me that I cannot use classes (for compatibility purposes, I guess). If in this branch I could use ES6 classes I'll definitively change it :) Let me check ;)

Reply:
![image](https://user-images.githubusercontent.com/13763335/177030723-8656875d-e875-45a1-8be3-e295fbeb07f5.png)

Using ES6 class I'm getting this linting error currently. In this version, it seems that I can use the class but maybe have I to define differently the async method?

Reply:
Hum or you need to upgrade the eslint module and rules ^^.

Reply:
You are right! Thanks, I didn't know I could update the eslint module (this is my first PR so sorry for first-time errors XD). I have pushed a new commit :)